packages/@dcl/ecs/src/composite/types.ts

```diff
@@ -0,0 +1,15 @@
+import { Entity } from '../engine/entity'
+import { JsonSchemaExtended } from '../schemas/ISchema'
+
+export type Composite = {
+  id: string
+  components: {
+    name: string
+    schema?: JsonSchemaExtended
+    data: Map<Entity, unknown>
+  }[]
+}
+
+export type CompositeProvider = {
+  getCompositeOrNull: (id: string) => Composite | null
```

Comment:
If loading composites from the deployed scene is going to be a thing, I recommend to converge towards a Promise from moment zero
```suggestion
  getCompositeOrNull: (id: string) => Promise<Composite> | null
```